onnx/test/version_converter_test.py

```diff
@@ -28,6 +28,9 @@ def _converted(
         # print(type(orig_model))
         converted_model = onnx.version_converter.convert_version(orig_model,
                 target_version)
+        print(str(orig_model))
+        print("---------")
+        print(str(converted_model))
```

Comment:
nit: Do you keep these print functions intentionally for debugging? Thank you for the resubmitting.

Reply:
@jcwchen Thanks! I forgot to remove them.